packages/mockserver-config-writer/src/mockserver-config/ui5-mock-yaml.ts

```diff
@@ -37,13 +37,18 @@ export async function enhanceYaml(
     let mockConfig;
     const manifest = fs.readJSON(join(webappPath, 'manifest.json')) as Partial<Manifest> as Manifest;
     const mockserverPath = config?.path || getMainServiceDataSource(manifest)?.uri;
+    const annotationSource = Object.values(getODataSources(manifest, ODataSourceType.ODataAnnotation));
+    const annotationsConfig = annotationSource.map((annotation) => ({
+        localPath: annotation.settings?.localUri ? `./webapp/${annotation.settings.localUri}` : '',
```

Comment:
I would do a join here in case there is anything funny in the localURI;
```
join('./webapp/', ${annotation.settings.localUri})
```

Reply:
And this needs to be read from `FolderTypes.WEBAPP` i.e. 
```
join(FolderTypes.WEBAPP, ${annotation.settings.localUri})
```

Reply:
should, I leave extra logic to check with `??`, if its undefined defaults to `' '`
And if it is, should I cover for it unit tests as well

Reply:
Ok! Just update it to
```
join('./webapp/', ${annotation.settings.localUri})
```
Thanks

Reply:
So if it defaults to `` then the `localPath` property will be added to the annotation config, if its undefined it wont be added. So the question is, what happens if the `localPath` is not present when the local mock server spins up?